newIDE/app/src/EventsFunctionsExtensionEditor/index.js

```diff
@@ -63,6 +63,7 @@ type Props = {|
   ) => void,
   onBehaviorEdited?: () => Promise<void>,
   onObjectEdited?: () => Promise<void>,
+  onFunctionEdited?: (extension: gdEventsFunctionsExtension) => Promise<void>,
```

Comment:
Is passing the extension mandatory? It seems not used by the parent, because the parent already knows the extension being edited (because by definition of the parent, it is the one telling the child which extension it is editing).